### PR TITLE
Add `MWC 4K 2024` wiki article, `osu!mania 4K World Cup 2024: Registrations Now Open!` news article

### PR DESCRIPTION
**Pull request by LeoFLT**
🎹

### CODE REVIEW

File: news/2024/2024-07-18-osu-mania-4k-world-cup-2024-registrations-now-open.md [hunk 1]
```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "osu!mania 4K World Cup 2024: Registrations Now Open!"
+date: 2024-07-18 10:00:00 +0000
+---
+
+Calling all key smashers – the osu!mania 4K World Cup is back for 2024!
```
Comment (Walavouchey) [inline news/2024/2024-07-18-osu-mania-4k-world-cup-2024-registrations-now-open.md:7]:
```suggestion
Calling all key smashers — the osu!mania 4K World Cup is back for 2024!
```

File: news/2024/2024-07-18-osu-mania-4k-world-cup-2024-registrations-now-open.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+
+![](/wiki/Tournaments/MWC/2024_4K/img/mwc4k2024-banner.jpg)
+
+Following our latest entry on the official tournament calendar ([Lazer Grand Arena](https://osu.ppy.sh/home/news/2024-06-18-lazer-grand-arena-registrations-now-open), if you haven't checked it yet), comes our yearly patented 4 key smashing competition. There are no major changes in the format for this year's osu!mania 4K World Cup, as the mappoolers are focusing on refining the current structure instead, lazer-focused (pun intended) on bringing the highest levels of awesomeness possible to viewers and competitors alike.
```
Comment (Walavouchey) [inline news/2024/2024-07-18-osu-mania-4k-world-cup-2024-registrations-now-open.md:11]:
```suggestion
Following our latest entry on the official tournament calendar ([Lazer Grand Arena](https://osu.ppy.sh/home/news/2024-06-18-lazer-grand-arena-registrations-now-open), if you haven't checked it yet) comes our yearly patented 4 key smashing competition. There are no major changes in the format for this year's osu!mania 4K World Cup, as the mappoolers are focusing on refining the current structure instead, lazer-focused (pun intended) on bringing the highest levels of awesomeness possible to viewers and competitors alike.
```

File: news/2024/2024-07-18-osu-mania-4k-world-cup-2024-registrations-now-open.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+
+- **1st:** 50% of the prize pool, single-run merch, profile badge, **osu!mania 4K Champion** user title for one year
+- **2nd:** 30% of the prize pool, single-run merch, profile badge
+- **3rd:** 20% of the prize pool, single-run merch, profile badge
+
+## How do I register?
+
+**[Click here!](https://osu.ppy.sh/community/tournaments/46)**
+
+Please note that teams will be formed by each country's captain *after* the registration phase has concluded. **The deadline for registering is August 01, 23:59 UTC. Make sure to submit yours before then!**
```
Comment (Walavouchey) [inline news/2024/2024-07-18-osu-mania-4k-world-cup-2024-registrations-now-open.md:41]:
```suggestion
Please note that teams will be formed by each country's captain *after* the registration phase has concluded. **The deadline for registering is August 1, 23:59 UTC. Make sure to submit yours before then!**
```

File: wiki/Tournaments/MWC/2024_4K/en.md [hunk 4]
```diff
@@ -0,0 +1,198 @@
...
+
+### Tournament rules
+
+1. The osu!mania 4K World Cup is a country-based 3 versus 3 team tournament, played on the osu!mania game mode (4K variant).
+   - Players from countries or territories (as seen on their profile flag) that are under a protectorate or sovereign state may choose to play for that country **if** the country listed on their profile is unable to field enough players to form a valid team. Refer to [this article](https://wikipedia.org/wiki/Dependent_territory#Lists_of_dependent_territories) for an overview of the criteria adopted. Players that have participated under another country previously may continue to do so.
+2. Beatmap scoring is based on ScoreV2.
+3. The minimum size for a team is 4 players, and the maximum is 6.
+4. The beatmap list for each round will be announced by the mappool selectors on the official stream, on the Sunday before the matches take place.
+5. The match schedules for each round will be announced by the tournament managers on this page, as well as the information sheet, on the Sunday before the matches take place.
+6. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins are allowed.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2024_4K/en.md:70]: "Use of the Visual Settings ... is allowed"

```suggestion
6. Use of the Visual Settings to alter background dim or disable beatmap elements like storyboards and skins is allowed.
```

File: wiki/Tournaments/MWC/2024_4K/en.md [hunk 5]
```diff
@@ -0,0 +1,198 @@
...
+   - Custom skin elements must not be used to alter core gameplay elements or mechanics in unintended ways.
+7. If less than the minimum amount of required players are present at match time, the match can be postponed for up to 10 minutes. If after this period there are still not enough players for either team, a *win by default* will be declared for the side with the most members present.
+   - The minimum amount of required players is the amount of players needed to play a beatmap without any vacant spots in the lobby (i.e. at least 3 participants from each team must be present for the match to begin).
+8. Exchanging players between games is allowed without limitations.
+9. If a game ends in a draw, it will be nullified and the beatmap will be replayed.
+10. Teams may ask for a rematch if a team member encounters technical issues while playing within 30 seconds or 25% of the beatmap drain length (whichever happens first) of the game's start. Referees may do rematch independently of either team if reasonable evidence of a disconnect/tech issue exists. Referees may veto any rematch requests at their discretion.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2024_4K/en.md:76]:
```suggestion
10. Teams may ask for a rematch if a team member encounters technical issues while playing within 30 seconds or 25% of the beatmap drain length (whichever happens first) of the game's start. Referees may perform a rematch independently of either team if reasonable evidence of a disconnect/tech issue exists. Referees may veto any rematch requests at their discretion.
```

File: wiki/Tournaments/MWC/2024_4K/en.md [hunk 6]
```diff
@@ -0,0 +1,198 @@
...
+      - Screenshots from the results screen must clearly show the affected player's score. This is the preferred method.
+      - Screenshots taken in-game at the time of disconnection may be accepted. Note that this method does not provide a one-to-one representation of that player's score. Using this method is not encouraged and it may be denied at the referee's discretion if the information provided is not sufficient to identify the player/score.
+      - All screenshots **MUST** be taken using the game itself (using `Shift` + `F12`), that is, they must be hosted on the `https://osu.ppy.sh/` domain. Any other form of screenshot will be denied.
+    - Player scores may be derived from the official stream as a last resort, in cases where the match is streamed.
+12. Players are expected to keep the match running fluently and without delays. Excessive match delays from the players' side may result in penalties being applied by the tournament managers. Disrupting the match by foul play, insulting or provoking other players or staff, delaying the match, or other deliberate inappropriate misbehaviour are strictly prohibited, and will be punished accordingly.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2024_4K/en.md:88]:
```suggestion
12. Players are expected to keep the match running fluently and without delays. Excessive match delays from the players' side may result in penalties being applied by the tournament managers. Disrupting the match by foul play, insulting and provoking other players or staff, delaying the match, or other deliberate inappropriate misbehaviour are strictly prohibited, and will be punished accordingly.
```

File: wiki/Tournaments/MWC/2024_4K/en.md [hunk 7]
```diff
@@ -0,0 +1,198 @@
...
+4. Each mappool has one tiebreaker, except for the Qualifiers.
+5. Possible mod choices for all beatmaps are [Hidden](/wiki/Gameplay/Game_modifier/Hidden), [Fade In](/wiki/Gameplay/Game_modifier/Fade_In), [Flashlight](/wiki/Gameplay/Game_modifier/Flashlight), and [Mirror](/wiki/Gameplay/Game_modifier/Mirror).
+6. Failed scores will be counted in their integrity. To that end, the [No Fail](/wiki/Gameplay/Game_modifier/No_Fail) mod will be enforced on every beatmap.
+   - Teams that purposefully disregard this instruction and team fail as a result will lose said game.
+7. The tiebreaker will be played under FreeMod conditions.
+
+### Scheduling information
+
+1. Each stage will be held on **a single weekend**.
+2. Matches in Qualifiers will be held during specific time slots proposed by the tournament managers. Each team may to choose a time slot in which they want to play.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2024_4K/en.md:188]:
```suggestion
2. Matches in Qualifiers will be held during specific time slots proposed by the tournament managers. Each team may choose a time slot in which they want to play.
```

File: wiki/Tournaments/MWC/2024_4K/en.md [hunk 8]
```diff
@@ -0,0 +1,198 @@
...
+   - **A team that does not pick a time slot by August 15, 23:59 UTC will be considered disqualified.**
+3. All bracket stages will be held between Saturday 00:00 and Sunday 19:00 UTC.
+   - Other times may be available upon request, consult with the tournament managers for more information.
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2024_4K/en.md:191]: comma splice

```suggestion
   - Other times may be available upon request. Consult with the tournament managers for more information.
```

File: wiki/Tournaments/MWC/2024_4K/en.md [hunk 9]
```diff
@@ -0,0 +1,198 @@
...
+
+- **[Sign up here](https://osu.ppy.sh/community/tournaments/46)**
+- [Livestream](https://www.twitch.tv/osulive)
+- [Discussion thread](https://osu.ppy.sh/community/forums/topics/1950284)
+
+## Ruleset
+
+### Tournament rules
+
+1. The osu!mania 4K World Cup is a country-based 3 versus 3 team tournament, played on the osu!mania game mode (4K variant).
```
Comment (Walavouchey) [inline wiki/Tournaments/MWC/2024_4K/en.md:64]: not sure why this is removed compared to last year

```suggestion
1. The osu!mania 4K World Cup is a country-based 3 versus 3 double-elimination team tournament, played on the osu!mania game mode (4K variant).
```

**Review by Walavouchey — CHANGES_REQUESTED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**